message-index/messages/GHC-1629/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Too many type arguments in constructor pattern
+summary: You have applied to many types to a constructor in a pattern.
```

Comment:
```suggestion
summary: Too many type applications to a constructor in a pattern.
```

Move to a passive tone to match the rest of the examples and be more specific that these are type applications.